src/modules/registrypreview/RegistryPreviewUI/Strings/en-US/Resources.resw

```diff
@@ -161,36 +161,34 @@
   </data>
   <data name="OkButtonText" xml:space="preserve">
     <value>OK</value>
+    <comment>as on the OK button</comment>
   </data>
   <data name="OpenButton.Label" xml:space="preserve">
-    <value>Open file...</value>
+    <value>Open…</value>
   </data>
   <data name="OpenDialogTitle" xml:space="preserve">
-    <value>Open Registry file...</value>
+    <value>Open Registry file</value>
   </data>
   <data name="RefreshButton.Label" xml:space="preserve">
-    <value>Reload from file</value>
+    <value>Reload</value>
   </data>
   <data name="RegistryButton.Label" xml:space="preserve">
-    <value>Open Registry Editor...</value>
+    <value>Open Registry editor</value>
```

Comment:
I disagree with this one, only because the application's name is cased with a capital E for Editor..